### PR TITLE
Voissa Add new extractor

### PR DESCRIPTION
**Pull request by tiburce**
Add Voissa support

### CODE REVIEW

**Comment by tiburce** [comment]
This site lag sometimes ... the script work well.

File: docs/supportedsites.md [hunk 1]
```diff
@@ -636,6 +636,7 @@
  - **vlive**
  - **Vodlocker**
  - **VoiceRepublic**
+ - **Voissa**
```
Comment (dstftw) [inline docs/supportedsites.md]: This is an autogenerated file.

**Comment by tiburce** [comment]
Sorry ,  I know now ...

File: youtube_dl/extractor/voissa.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        }
+    }
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        video_id = mobj.group('id')
+        display_id = mobj.group('display_id')
+        webpage = self._download_webpage(url, video_id)
+
+        url_info = self._download_webpage('http://communaute.voissa.com/playlist.php?id=' + str(video_id), 'Url inf')
```
Comment (dstftw) [inline youtube_dl/extractor/voissa.py]: Use `self._download_xml` instead and process as XML.

File: youtube_dl/extractor/voissa.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        video_url = self._search_regex(r'<media:content url="([^"]+)"', url_info, 'video URL')
+        thumbnail = self._search_regex(r'<media:thumbnail url="([^"]+)"', url_info, 'thumbnail URL', fatal=False)
+
+        title = self._search_regex(r'<title>Vidéo ([^.]+).+Voissa</title>', webpage, 'title')
+
+        mobj = re.search(r'<span itemprop="name">(?P<uploader>[^<]+)</span>.+vu (?P<viewcount>\d+)', webpage)
+        uploader = mobj.group('uploader')
+        view_count = mobj.group('viewcount')
```
Comment (dstftw) [inline youtube_dl/extractor/voissa.py]: `view_count` must be int.

**Pull request closed without merging**